### ISSUE TITLE
update list function with toList

### ISSUE THREAD

**Issue by jaun-rg** [terraform-aws-modules/terraform-aws-eks#1394]
## Description

using the last terraform version (0.15.4), this module present errors associated to list functions (list function is no longer available)

## Versions

 - Terraform: 0.15.4

 - Provider(s): 
    + provider registry.terraform.io/hashicorp/aws v3.36.0
    + provider registry.terraform.io/hashicorp/kubernetes v2.0.3
    + provider registry.terraform.io/hashicorp/local v2.1.0
    + provider registry.terraform.io/hashicorp/null v3.1.0
    + provider registry.terraform.io/hashicorp/random v3.1.0
    + provider registry.terraform.io/hashicorp/template v2.2.0

 - Module: 16.2.0

## Reproduction

use 
terraform plan

## Expected behavior

the plan output

## Actual behavior

A list of errors associated to list function

### Terminal Output Screenshot(s)

```bash
Error: Error in function call
│
│   on .terraform/modules/api.eks/outputs.tf line 3, in output "cluster_id":
│    3:   value       = element(concat(aws_eks_cluster.this.*.id, list("")), 0)
│
│ Call to function "list" failed: the "list" function was deprecated in
│ Terraform v0.12 and is no longer available; use tolist([ ... ]) syntax to
│ write a literal list.
╵
╷
│ Error: Error in function call
│
│   on .terraform/modules/api.eks/outputs.tf line 11, in output "cluster_arn":
│   11:   value       = element(concat(aws_eks_cluster.this.*.arn, list("")), 0)
│
│ Call to function "list" failed: the "list" function was deprecated in
│ Terraform v0.12 and is no longer available; use tolist([ ... ]) syntax to
│ write a literal list.
╵
╷
│ Error: Error in function call
│
│   on .terraform/modules/api.eks/outputs.tf line 16, in output "cluster_certificate_authority_data":
│   16:   value       = element(concat(aws_eks_cluster.this[*].certificate_authority[0].data, list("")), 0)
│
│ Call to function "list" failed: the "list" function was deprecated in
│ Terraform v0.12 and is no longer available; use tolist([ ... ]) syntax to
│ write a literal list.
╵
╷
│ Error: Error in function call
│
│   on .terraform/modules/api.eks/outputs.tf line 21, in output "cluster_endpoint":
│   21:   value       = element(concat(aws_eks_cluster.this.*.endpoint, list("")), 0)
│
│ Call to function "list" failed: the "list" function was deprecated in
│ Terraform v0.12 and is no longer available; use tolist([ ... ]) syntax to
│ write a literal list.
╵
╷
│ Error: Error in function call
│
│   on .terraform/modules/api.eks/outputs.tf line 26, in output "cluster_version":
│   26:   value       = element(concat(aws_eks_cluster.this[*].version, list("")), 0)
│
│ Call to function "list" failed: the "list" function was deprecated in
│ Terraform v0.12 and is no longer available; use tolist([ ... ]) syntax to
│ write a literal list.
╵
╷
│ Error: Error in function call
│
│   on .terraform/modules/api.eks/outputs.tf line 61, in output "cloudwatch_log_group_name":
│   61:   value       = element(concat(aws_cloudwatch_log_group.this[*].name, list("")), 0)
│
│ Call to function "list" failed: the "list" function was deprecated in
│ Terraform v0.12 and is no longer available; use tolist([ ... ]) syntax to
│ write a literal list.
╵
╷
│ Error: Error in function call
│
│   on .terraform/modules/api.eks/outputs.tf line 66, in output "cloudwatch_log_group_arn":
│   66:   value       = element(concat(aws_cloudwatch_log_group.this[*].arn, list("")), 0)
│
│ Call to function "list" failed: the "list" function was deprecated in
│ Terraform v0.12 and is no longer available; use tolist([ ... ]) syntax to
│ write a literal list.
╵
```

**Comment by barryib** [comment]
Are you sure that you're using the module v16.2.0 ? 

```shell
Error: Error in function call
│
│   on .terraform/modules/api.eks/outputs.tf line 3, in output "cluster_id":
│    3:   value       = element(concat(aws_eks_cluster.this.*.id, list("")), 0)
│
│ Call to function "list" failed: the "list" function was deprecated in
│ Terraform v0.12 and is no longer available; use tolist([ ... ]) syntax to
│ write a literal list.
```

Because I don't where you're error is coming from https://github.com/terraform-aws-modules/terraform-aws-eks/blob/v16.2.0/outputs.tf#L3

This has been fixed (normally) in [v15.0.0](https://github.com/terraform-aws-modules/terraform-aws-eks/releases/tag/v15.0.0). Please see [changelog](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/CHANGELOG.md#v1500---2021-04-16).

**Comment by barryib** [comment]
Closing. Feel free to reopen it if needed.
